nixos/modules/system/activation/activation-script.nix

```diff
@@ -60,7 +57,7 @@ let
       exit $_status
     '';
 
-  path = with pkgs; map getBin
+  path = with pkgs; map lib.getBin
```

Comment:
somehow I had to do this manually...

Reply:
Yeah, I think your script has an error when there's an inner `with`.